extra_data/tests/test_run_files_map.py

```diff
@@ -69,6 +70,9 @@ def map_paths_for_run(self, directory):
     assert isinstance(file_info['train_ids'], np.ndarray)
     assert isinstance(file_info['control_sources'], frozenset)
     assert isinstance(file_info['instrument_sources'], frozenset)
+    
+    fc = get_global_filecache()
+    fc.force_close(extra_file)
```

Comment:
If closing files is working correctly, this addition shouldn't be necessary any more, right?

Reply:
In this particular case, of course, the files are closed. As we discussed, we cannot guarantee that files will be actually closed in all cases (e.g. many `DataCollections`). Thus, the good practice is to close them explicitly anyway. I prefer to keep as is.

This code is also documentation, kind of a reminder.